lnrpc/marshall_utils.go

```diff
@@ -56,6 +60,10 @@ func UnmarshallAmt(amtSat, amtMsat int64) (lnwire.MilliSatoshi, error) {
 		return 0, ErrSatMsatMutualExclusive
 	}
 
+	if amtSat < 0 || amtMsat < 0 {
+		return 0, ErrNegativeFeeLimit
```

Comment:
why should the `UnmarshallAmt` func know that in case the value is negative that it relates to the feeLimit ?